approval_polls/views.py

```diff
@@ -36,6 +37,12 @@ class DetailView(generic.DetailView):
   def get_queryset(self):
       return Poll.objects.filter(pub_date__lte=timezone.now())
 
+@login_required
+def delete_poll(request):
```

Comment:
@ashimaathri 
I am not sure if I understand what you mean.
I've checked the Django document ans it seems like detail view has no delete() 
https://docs.djangoproject.com/en/1.8/ref/class-based-views/flattened-index/#detailview
Also, do you want me to create a new view or move the delete function to an existing view?
Thanks!